TicTacToe/TicTacToe/WinConditions.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TicTacToe {
+
+    public class WinConditions {
+
+        public bool IsWinner(string[] gameBoard) {
+            int boardDimension = (int) Math.Sqrt(gameBoard.Length);
+            return IsWin(GetRows(boardDimension, gameBoard)) || IsWin(GetColumns(boardDimension, gameBoard)) ||
+                IsWin(GetDiagonals(boardDimension, gameBoard));
+        }
+
+        private bool IsWin(List<List<string>> rows) {
```

Comment:
should this really be called 'rows'? I think a more general name might fit better here